packages/node/tsconfig.json

```diff
@@ -13,6 +13,7 @@
   },
   "include": [
     "src/**/*",
+    "src/.api/**/*",
```

Comment:
nit: is the not redundant when we already have `src/**/*`?

Reply:
@kanadgupta added this so I'll let him chime in with more context!

Reply:
i _think_ this is because tsconfig ignores directories that start with a period by default, but let me double check that. this may have been an accidental holdover from when i was getting the SDK up and running

Reply:
yeah looks like those directories aren't included by default, see https://github.com/microsoft/TypeScript/issues/49555